ggshield/cmd/honeytoken/create.py

```diff
@@ -0,0 +1,97 @@
+import random
+import string
+import sys
+from typing import Any, Dict, Optional
+
+import click
+from pygitguardian import GGClient
+from pygitguardian.models import Detail, HoneytokenResponse
+
+from ggshield.cmd.common_options import add_common_options
+from ggshield.core.client import create_client_from_config
+from ggshield.core.config import Config
+from ggshield.core.errors import UnexpectedError
+from ggshield.core.utils import urljoin
+
+
+def _generate_random_honeytoken_name() -> str:
+    """Generate a honeytoken name base on random string of six alphanumeric characters"""
+    letters_and_digits = string.ascii_letters + string.digits
+    random_str = "".join(random.choice(letters_and_digits) for i in range(8))
+    return f"ggshield-{random_str}"
```

Comment:
Should it be ggtoken or something like that ?

Reply:
the goal is to know that this honeytokens have been created via ggshield